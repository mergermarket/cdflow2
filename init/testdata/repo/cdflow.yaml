version: 2
config:
  image: mergermarket/cdflow2-config-acuris
  params:
    team: %{team}
    org: %{org}
terraform: mergermarket/terraform:1.2.1
