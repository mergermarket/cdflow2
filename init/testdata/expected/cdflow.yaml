version: 2
config:
  image: mergermarket/cdflow2-config-acuris
  params:
    team: platform
    org: ION
terraform: mergermarket/terraform:1.2.1
